README.md

```diff
@@ -134,51 +134,50 @@ case class Success(result: BigDecimal) extends Calculation
 
 There are several options to map that to definitions in Protocol Buffers and each has some mismatch.
 Protofiles know the concept of `oneof` which is more a *union* of types than a type hierarchy.
-Therefore a `oneof` named `calculation` with options `error` and `success` cannot be reused. The `oneof` must be
-defined in each position a `Calculation` occurs.
+Therefore a `oneof` named `calculation` with options `error` and `success` cannot be reused.
+The `oneof` must be defined in each position a `Calculation` occurs.
 
-We came up with this corresponding Protofile definition:
+Take for example the following Protofile definition:
 
 ```
-message Calculation {
+message Result {
   message Error {
     String message = 1;
   }
   message Success {
     String result = 1;
   }
-  oneof value {
+  oneof Calculation {
```

Comment:
We can restore support for the `value` pattern later by enabling the mapping between a message with a single field (`value` and the type of that field). That would also make it easier to support value classes. But there is nothing special on the Protobuf side about naming this `value` and in fact we see more examples where this pattern is **not** followed in MOIA. We also don't have a linter to enforce it.